client/src/routes.js

```diff
@@ -9,11 +12,21 @@ const routes = [
     name: 'Home',
     component: Home,
   },
+  {
+    path: '/browse',
+    name: 'Recipes',
+    component: Browse,
+  },
   {
     path: '/auth',
     name: 'Auth',
     component: Auth,
   },
```

Comment:
nitpick: can you rearrange these so browse is under auth so that browse and user recipes are next to eachother